frontend/src/metabase/static-viz/components/XYChart/Values/Values.tsx

```diff
@@ -112,13 +134,15 @@ export default function Values({
 
   return (
     <>
-      {verticalOverlappingFreeValues.map((singleSerieValues, seriesIndex) => {
-        const compact = getCompact(singleSerieValues.map(value => value.datum));
+      {verticalOverlappingFreeValues.map((singleSeriesValues, seriesIndex) => {
```

Comment:
just a rename from `singleSerieValues` (serie without an s) to `singleSeriesValues